compiler-rt/lib/memprof/memprof_new_delete.cpp

```diff
@@ -43,6 +43,7 @@ enum class align_val_t : size_t {};
     ReportOutOfMemory(size, &stack);                                           \
   return res;
 
+#if !SANITIZER_APPLE
```

Comment:
How do operator new and delete get intercepted on Apple?

Also, are OPERATOR_NEW_BODY* and OPERATOR_DELETE_BODY* macros needed if these interceptors are not included - can the whole thing be put under "#if !SANITIZER_APPLE"?